src/components/common/navbar.tsx

```diff
@@ -36,16 +39,27 @@ const Navbar: FC<Props> = ({
     setOpenSearchField(false);
   };
 
+  const onSearch = (event: KeyboardEvent<HTMLInputElement>) => {
+    if (event.key === "Enter") {
+      event.preventDefault();
+      navigate(`/orderDetails/${searchValue}`);
+    }
+  };
+
+  const toggleShowQuickLinks = () => {
+    setShowQuickLinks(!showQuickLinks);
```

Comment:
for single line methods, keep it inline.
DOn't need to define a function and invoke it, if the method is just one line long.

Reply:
done